System.IO.Ports/SerialDataEvent.cs

```diff
@@ -7,7 +7,8 @@ namespace System.IO.Ports
 {
     internal class SerialDataEvent : BaseEvent
     {
-        public int SerialDeviceIndex;
-        public SerialData Event;
+        public int SerialDeviceIndex { get; set; }
```

Comment:
Both of these need to be fields. Won't work if they are properties. This is comming from the Events library and native code, is not something that can be changed easily.